src/Cake.Core/Scripting/ScriptConventions.cs

```diff
@@ -85,6 +85,19 @@ public IReadOnlyList<Assembly> GetDefaultAssemblies(DirectoryPath root)
             // This is just to please Roslyn when running under Mono. See issue https://github.com/dotnet/roslyn/issues/19364
             result.Add(_loader.Load(new AssemblyName("System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"))); // System.Runtime
             result.Add(_loader.Load(new AssemblyName("System.Collections, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"))); // System.Collections
+
+            try
+            {
+                result.Add(_loader.Load(new AssemblyName("netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51"))); // NETStandard.Library
+            }
+            catch
+            {
+                /*
+                 * Silently continue instead and let it blow up during runtime if netstandard assembly was not found.
+                 * TODO: Log that netstandard assembly was not found.
+                 * Unfortunately, logger is not available in this class, and that would be too big of a change for the 0.26.1 hotfix release.
```

Comment:
Is it really that big change to inject the logger and write the message here?

Reply:
Well, class is public and constructor also. A new `internal` constructor however could be done. As this is for a hotfix, only internal changes are allowed.

Reply:
Ah, ok.